.pipelines/e2e.yml

```diff
@@ -45,6 +45,7 @@ jobs:
   - template: ./templates/template-run-rp-and-e2e.yml
     parameters:
       workingDirectory: ${{ variables.modulePath }}
+      aroE2ETags: $(ARO_E2E_TAGS)
```

Comment:
Each and every time we introduce a configurable, my questions is "how often you gonna change this"? Why not just use `test=e2e` and just leave it forever?

Reply:
Yea, I was not sure about this, so I made it configurable.